packages/server/lib/cloud/protocol.ts

```diff
@@ -28,6 +29,8 @@ const setupProtocol = async (url?: string): Promise<AppCaptureProtocolInterface
       console: 'inherit',
       sandbox: {
         Debug,
+        performanceNow: performance.now,
+        performanceTimeOrigin: performance.timeOrigin,
```

Comment:
are we able to pass in a performance object?
```
performance: {
  now: performance.now,
  timeOrigin: performance.timeOrigin,
}
```

Reply:
Updated: [`a30e249` (#26679)](https://github.com/cypress-io/cypress/pull/26679/commits/a30e24934b52cd890152e0868684497d55879974)